lib/swoosh/adapters/mail_place.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Swoosh.Adapters.MailPace do
```

Comment:
This file is called `mail_place.ex` but should be `mail_pace.ex`.

Reply:
Yup, my bad